tau-ml-env.yaml

```diff
@@ -8,8 +8,11 @@ dependencies:
   - absl-py=0.12.0=pyhd8ed1ab_0
   - afterimage=1.21=ha9998ff_1003
   - aiohttp=3.7.4=py38h497a2fe_0
+  - alembic=1.4.1=py_0
```

Comment:
This comment unrelated with the following PR but I have problem installing the environment from this yaml. When I do `conda env create -f tau-ml-env.yaml` I obtain the list of corresponding errors:

CondaVerificationError: The package for cudnn located at /nfs/dust/cms/user/mykytaua/softML/miniconda3/pkgs/cudnn-8.1.0.77-h90431f1_0
appears to be corrupted. The path 'include/cudnn.h'
specified in the package manifest cannot be found.

CondaVerificationError: The package for cudnn located at /nfs/dust/cms/user/mykytaua/softML/miniconda3/pkgs/cudnn-8.1.0.77-h90431f1_0
appears to be corrupted. The path 'include/cudnn_adv_infer.h'
specified in the package manifest cannot be found.
...

Reply:
Strange, I tested the fresh installation of the environment (`conda env create -n tau-ml-dev -f tau-ml-env.yaml`) and didn't see any errors... Do you still observe it if you try to install the environment from the yaml which is currently on the master branch (i.e. before me adding new packages)?

Reply:
For me, it does not work either

Reply:
@yaourtpourtoi are you testing it on naf.desy or lxplus?

Reply:
ok, the issue it closed. It is working after cleaning of the conda base environment